src/drivers/intel/cavs/ssp.c

```diff
@@ -967,13 +967,13 @@ const struct dai_driver ssp_driver = {
 	.dma_caps = DMA_CAP_GP_LP | DMA_CAP_GP_HP,
 	.dma_dev = DMA_DEV_SSP,
 	.ops = {
-		.trigger		= ssp_trigger,
-		.set_config		= ssp_set_config,
-		.pm_context_store	= ssp_context_store,
-		.pm_context_restore	= ssp_context_restore,
-		.get_handshake		= ssp_get_handshake,
-		.get_fifo		= ssp_get_fifo,
-		.probe			= ssp_probe,
-		.remove			= ssp_remove,
+		.trigger = ssp_trigger,
+		.set_config = ssp_set_config,
+		.pm_context_store = ssp_context_store,
+		.pm_context_restore = ssp_context_restore,
+		.get_handshake = ssp_get_handshake,
+		.get_fifo = ssp_get_fifo,
+		.probe = ssp_probe,
+		.remove = ssp_remove,
```

Comment:
Pass-by coding style change? Should be separate commit, although I'm not sure this particular change is even needed.

Reply:
I've made it accidentally. It is corrected now.